footer attempt 2

```diff
@@ -0,0 +1,154 @@
+        initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Document</title>
+    </head>
+    <body>
+        <div class="footer-container">
+            <div class="footer-heading-footer-1">
+                <h2>About us</h2>
+                <a href="#">Feedback</a>
+                <a href="#">Reviews</a>
+                <a href="#">Staff</a>
+                <a href="#">Support</a>
+                <a href="#">Terms of Service</a>
+            </div>
+            <div class="footer-heading-footer-2">
+                <h2>Contact</h2>
+                <a href="#">Applications</a>
+                <a href="#">Openings</a>
+                <a href="#">Contact</a>
+                <a href="#">Support</a>
+                <a href="#">Sponserships</a>
+            </div>
+            <div class="footer-heading-footer-3">
+                <h2>Social Media</h2>
+                <a href="#">Instagram</a>
+                <a href="#">Youtube</a>
+                <a href="#">Facebook</a>
+                <a href="#">Twitter</a>
+            </div>
+            <div class="footer-email-form">
+                <h2>Join our newsletter</h2>
+                <input type="email" placeholder="Enter 
+                your email adress" id="footer-email">
+                <input type="submit" value="Sign-up"
+                id="footer-email-btn">
+            </div>
+        </div>
+    </body>
+    </html>
+</html>
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.footer-container {
+    background-color: lightsteelblue;
+    padding: 2rem 0 4rem 0;
+}
+
+.footer {
+    width: 80%;
+    height: 40vh;
+    background-color: lightsteelblue;
+    color: lightsteelblue;
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap; 
+    margin: 0 auto;
+}
+
+.footer-heading {
+    display: flex;
+    flex-direction: column;
+    margin-right: 4rem;
+
+}
+
+.footer-heading h2 {
+    margin-bottom: 2rem;
+}
+
+.footer-heading * {
+    color: lightsteelblue;
+    text-decoration: none;
+    margin-bottom: 0.5rem;
+}
+
+.footer-heading a:hover {
+    color: lightsteelblue; 
+    transition: 0.3s ease-out;
+}
+
+.footer-email-form h2 {
+    margin-bottom: 2rem;
+    
+}
+
+#footer-email {
+    width: 250px;
+    height: 40px;
+    border-radius: 4px; 
+    outline: none;
+    border: none;
+    padding-left: 0.5rem;
+    font-size: 1rem;
+    margin-bottom: 1rem;
+}
+
+#footer-email::placeholder {
+    color: lightsteelblue;
+}
+
+#footer-email-btn {
+    width: 100px;
+    height: 40px;
+    border-radius: 4px;
+    background-color: lightsteelblue;
+    outline: none;
+    border: none;
+    color: mintcream;
+    font-size: 1rem;
+}
+
+#footer-email-btn:hover {
+    cursor: pointer;
+    background-color: maroon;
+    transition: all 0.4s ease-out;
+}
+
+@media screen and (max-width: 1150px) {
```

Comment:
Curious, did you learn about media queries?

Reply:
actually i followed along on a tutorial to have a base footer. and if anything i can try to find a work around or some other possible solution(s)

Reply:
Got it. That's ok. I think you just need to move these into a separate CSS file and update the HTML.